src/hyperion/parameters/plan_specific/wait_for_robot_load_then_center_params.py

```diff
@@ -96,6 +96,11 @@ def _preprocess_hyperion_params(
         all_params["num_images_per_trigger"] = 1
         all_params["trigger_mode"] = TriggerMode.FREE_RUN
         all_params["upper_left"] = np.zeros(3, dtype=np.int32)
+        all_params["expected_energy_ev"] = (
+            experiment_params.requested_energy_kev * 1000
+            if experiment_params.requested_energy_kev
+            else None
+        )
```

Comment:
Should: Let's have a discussion (probably with @dperl-dls) to work out if we can come up with a nicer way of using the real energy if one is not requested. Having the logic split between here and the plan isn't great